### DISCUSSION TITLE
Uploaded documents are getting removed if DocsGPT instance is restarted.

### DISCUSSION THREAD

**Discussion by Vipin525** [arc53/DocsGPT#866] [Q&A] [answered]
Hi,

I have hosted the DocsGPT locally on cloud and its running perfectly, but if its restarted the running pod/container then uploaded documents are also getting removed, if you could  tell how can this be avoided, will be great help.

Thanks,

**Comment by dartpain** [comment, marked as answer]
Seems like the issue is with your container deleting your old data. I would suggest you use a database vectorstore layer not faiss for production as it might cause issues.
Check out our config File as we support different vector databases. And when you deploy your vector db make sure that the instance will not be ephemeral.
